FGEGraphics/UISystem/UINumberSlider.cs

```diff
@@ -29,34 +30,53 @@ public class UINumberSlider : UIClickableElement
     /// <summary>The maximum slider value.</summary>
     public double Max;
 
-    /// <summary>Whether to use an integer grid instead of decimals.</summary>
-    public bool IsInt;
+    /// <summary>
+    /// The grid-snapping interval. 
+    /// Set to <c>0.0</c> or less for a gridless slider.
+    /// </summary>
+    public double Interval;
+
+    /// <summary>
+    /// Whether the slider should use integers instead of decimals.
+    /// If true, <see cref="Interval"/> should be <c>1.0</c>.
```

Comment:
What if I want an integer step of `2`... or `5`... or, yknow, any other integer?

ps `integral` is a weird word to use here, means a different thing in math usually. `IsInt` was clearer, just `Integer` would also be fine, or `Round`. Or just let it be defined by a Step (non-bypassable interval)